sealed_sector_health/health.go

```diff
@@ -0,0 +1,25 @@
+package sealed_sector_health
+
+// Health represents the healthiness of a sector managed by a
+// sector builder.
+type Health int
+
+const (
+	Unknown              Health = iota
```

Comment:
Why not just keep those in the sectorbuilder package, with a prefix - like `HealthUnknown`, `HealthOk`?

Reply:
Good question. My answer is more or less the same as to your earlier question: I don't want to have to manipulate the `const` names again if (when?) we add staged sector health to this API.